src/formats/SDF.cpp

```diff
@@ -339,43 +309,45 @@ void SDFFormat::write_next(const Frame& frame) {
 
 optional<uint64_t> SDFFormat::forward() {
     auto position = file_.tellpos();
-    size_t natoms = 0;
-    size_t nbonds = 0;
-    auto counts_line = string_view("");
-    try {
-        // Ignore junk lines
-        for (size_t i=0; i<3; i++) {
-            file_.readline();
-        }
 
-        if (file_.eof()) {
-            return nullopt;
-        }
+    // Ignore junk lines
```

Comment:
Very pedantic comment: every time I scroll past this comment I wonder what junk lines are. This are just two header lines that contain name and author information (https://pubs.acs.org/doi/pdf/10.1021/ci00007a012 section 3.1).